### PR TITLE
add only() to editor

### PR DESCRIPTION
**Pull request by elalish**
Follow-on to #330 - now we have OpenSCAD `#` functionality with `show()` and `*` functionality with `only()`. Ghosting out the final output gives nice context to the object shown with `only()`.

### CODE REVIEW

File: bindings/wasm/examples/editor.d.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 declare function show(manifold: Manifold): Manifold;
+declare function only(manifold: Manifold): Manifold;
```
Comment (pca006132) [inline bindings/wasm/examples/editor.d.ts:16]: Should we put the documentation for `show` and `only` here as well?

**Comment by pca006132** [comment]
Btw, wondering what would happen if there are multiple `only` within 1 model. The last one wins or it will be `only(union(...))`?

**Comment by elalish** [comment]
It'll include each `only` and `show` mesh separately. I think that's pretty close to OpenSCAD's behavior too.

File: bindings/wasm/examples/editor.d.ts (hunk 1 above)
Comment (elalish) [reply bindings/wasm/examples/editor.d.ts:32]: Good call, done.

**Pull request merged**